projects/github-actions/required-review/src/request-review.js

```diff
@@ -0,0 +1,42 @@
+const core = require( '@actions/core' );
+const github = require( '@actions/github' );
+const getUsername = require( './get-username.js' );
+
+/**
+ * Request review from the given team
+ *
+ * @param {string} team - GitHub team slug, or @ followed by a GitHub user name.
+ */
+async function requestReviewer( teams ) {
+	const octokit = github.getOctokit( core.getInput( 'token', { required: true } ) );
+	const owner = github.context.payload.repository.owner.login;
+	const repo = github.context.payload.repository.name;
+	const pr = github.context.payload.pull_request.number;
+
+	let userReviews = []
+	let teamReviews = []
+
+	for (var i = 0; i < teams.length; i++) {
+		const t = teams[ i ]
```

Comment:
I'd recommend doing it like this instead
```suggestion
	for ( const t of teams ) {
```